src/clients/AcrossConfigStoreClient.ts

```diff
@@ -298,6 +304,10 @@ export class AcrossConfigStoreClient {
           this.cumulativeRouteRateModelUpdates.push({ ...passedArgs, routeRateModel: {}, l1Token });
         }
       } catch (err) {
+        if (isDefined((err as Error)?.message)) {
```

Comment:
Do we have an isError function?

Reply:
> Do we have an isError function?

Added in https://github.com/across-protocol/sdk-v2/pull/220/commits/4e93db79e02b22d186998148ed1666e40e60e574